File: CS/ComboBoxLargeDataBase/Controllers/HomeController.cs
using System.Web.Mvc;
using ComboBoxLargeDataBase.Model;

namespace ComboBoxLargeDataBase.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return View(DataHelper.Persons);
        }
        public ActionResult ComboBoxPartial() {
            return PartialView("ComboBoxPartial", DataHelper.Persons);
        }
    }
}

File: VB/ComboBoxLargeDataBase/Controllers/HomeController.vb
Imports System.Web.Mvc
Imports ComboBoxLargeDataBase.Model

Namespace ComboBoxLargeDataBase.Controllers

    Public Class HomeController
        Inherits Controller

        Public Function Index() As ActionResult
            Return View(Persons)
        End Function

        Public Function ComboBoxPartial() As ActionResult
            Return PartialView("ComboBoxPartial", Persons)
        End Function
    End Class
End Namespace
